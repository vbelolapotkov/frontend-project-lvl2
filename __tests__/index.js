import genDiff from '../src/index.js';

test('plain json diff', () => {
  const diff = genDiff('./__fixtures__/file1.json', './__fixtures__/file2.json');
  expect(diff).toMatchSnapshot();
});
