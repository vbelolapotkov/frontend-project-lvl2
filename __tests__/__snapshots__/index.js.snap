// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`plain json diff 1`] = `
"{
  - follow: false
    host: hexlet.io
  - proxy: 123.234.53.22
  - timeout: 50
  + timeout: 20
  + verbose: true
}"
`;
